docs/sections/authorization.html

```diff
@@ -26,6 +26,23 @@ <h2>Authorization</h2>
     </li>
   </ul>
 
+  <section id="presentation-authentication">
+    <h3>Presentation Authentication</h3>
+    <p>
+      This API specification describes two ways in which presentations from a holder to a verifier can be authenticated: 
+    </p>
+    <ul>
+      <li>
+        DIDAuth, authenticated by proving a Verifiable Presentation signed over a challenge. 
+      </li>
+      <li>
+        OAuth, authenticated with an access token provided in the request header (with submit:presentations scope). In 
+        order to avoid double authentication a Presentation object is passed in the request body, not a Verifiable 
+        Presentation (i.e. without a VP proof element). 
```

Comment:
```suggestion
        OAuth, authenticated with an access token provided in the request header (with scope of `submit:presentations`).
        To avoid double authentication, a Presentation object &mdash; not a Verifiable Presentation, i.e., lacking
        a VP proof element &mdash; is passed in the request body.
```